ingestion/src/metadata/ingestion/ometa/mixins/lineage_mixin.py

```diff
@@ -185,7 +189,12 @@ def _create_lineage_by_table_name(
                 )
             else:
                 multiple_to_fqns = [to_entity]
-            if not from_entity or not to_entity:
+            if (
+                not multiple_to_fqns
+                or multiple_to_fqns == []
+                or not multiple_from_fqns
+                or multiple_from_fqns == []
```

Comment:
These conditions are redundant. It is the same as `if not multiple_to_fqns or not multiple_from_fqns`